hack/tool/tackle

```diff
@@ -374,6 +440,177 @@ class Tackle12Import:
+        for sh2 in collection:
+            # Prepare StakeHolder
+            sh             = Tackle2Object(sh2)
+            sh.name        = sh2['name']
+            sh.email       = sh2['email']
+            sh.groups      = sh2['stakeholderGroups']
+            sh.jobFunction = sh2['jobFunction']
+            self.add('stakeholders', sh)
+
+        ### STAKEHOLDER GROUPS ###
+        collection = apiJSON(self.tackle2Url + "/hub/stakeholdergroups", self.tackle2Token)
+        for shg2 in collection:
+            # Prepare StakeholderGroup
+            shg             = Tackle2Object(shg2)
+            shg.name        = shg2['name']
+            shg.description = shg2['description']
+            self.add('stakeholdergroups', shg)
+
+        ### JOB FUNCTION ###
+        collection = apiJSON(self.tackle2Url + "/hub/jobfunctions", self.tackle2Token)
+        for jf2 in collection:
+            # Prepare JobFunction
+            jf              = Tackle2Object(jf2)
+            jf.name         = jf2['name']
+            jf.stakeholders = jf2['stakeholders']
+            self.add('jobfunctions', jf)
+
+        ### BUSINESS SERVICE ###
+        collection = apiJSON(self.tackle2Url + "/hub/businessservices", self.tackle2Token)
+        for bs2 in collection:
+            bs              = Tackle2Object(bs2)
+            bs.name         = bs2['name']
+            bs.description  = bs2['description']
+            bs.owner        = bs2['owner']  # Stakeholder
+            self.add('businessservices', bs)
+
+        ### PROXY ###
+        collection = apiJSON(self.tackle2Url + "/hub/proxies", self.tackle2Token)
+        for pr2 in collection:
+            pr          = Tackle2Object(pr2)
+            pr.enabled  = pr2['enabled']
+            pr.kind     = pr2['kind']
+            pr.host     = pr2['host']
+            pr.port     = pr2['port']
+            pr.excluded = pr2['excluded']
+            pr.identity = pr2['identity']
+            self.add('proxies', pr)
+
+        ### IDENTITY ###
+        collection = apiJSON(self.tackle2Url + "/hub/identities", self.tackle2Token)
+        for id2 in collection:
+            id             = Tackle2Object(pr2)
+            id.kind        = id2['kind']
+            id.name        = id2['name']
+            id.description = id2['description']
+            id.user        = id2['user']
+            id.password    = id2['password']
+            id.key         = id2['key']
+            id.settings    = id2['settings']
+            self.add('identities', id)
+
+    def dumpTackle2Buckets(self):
+        for bucket in self.data['buckets']:
+            debugPrint("Downloading bucket data for %s from owner %s" % (bucket['path'], bucket['owner']))
+            bucketListing = apiRaw(self.tackle2Url + "/hub/" + bucket['owner'] + "/bucket/", self.tackle2Token)
+            bucketEntries = re.findall(("href=\"([^\"]+)\""), bucketListing)
```

Comment:
Not sure if there not a better way to list bucket content

Reply:
Converting to draft until ```buckets``` ednpoint is implemented in API and related code is updated.